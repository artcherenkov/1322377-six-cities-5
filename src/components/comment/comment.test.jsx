import React from 'react';
import renderer from 'react-test-renderer';

import Comment from './comment';

const comment = {
  comment: `hello it is comment`,
  date: `2020-11-24T16:15:25.719Z`,
  id: 1,
  rating: 3,
  user: {
    avatarUrl: `avatar url`,
    id: 1,
    isPro: true,
    name: `Alex`,
  }
};

it(`Comment is rendered correctly`, () => {
  const tree = renderer.create(
      <Comment comment={comment} />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
