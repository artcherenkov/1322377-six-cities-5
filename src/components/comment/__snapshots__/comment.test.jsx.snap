// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment is rendered correctly 1`] = `
<li
  className="reviews__item"
>
  <div
    className="reviews__user user"
  >
    <div
      className="reviews__avatar-wrapper user__avatar-wrapper"
    >
      <img
        alt="Reviews avatar"
        className="reviews__avatar user__avatar"
        height="54"
        src="avatar url"
        width="54"
      />
    </div>
    <span
      className="reviews__user-name"
    >
      Alex
    </span>
  </div>
  <div
    className="reviews__info"
  >
    <div
      className="reviews__rating rating"
    >
      <div
        className="reviews__stars rating__stars"
      >
        <span
          style={
            Object {
              "width": "60%",
            }
          }
        />
        <span
          className="visually-hidden"
        >
          Rating
        </span>
      </div>
    </div>
    <p
      className="reviews__text"
    >
      hello it is comment
    </p>
    <time
      className="reviews__time"
      dateTime="2020-11-24"
    >
      November 2020
    </time>
  </div>
</li>
`;
